db/migrate/20190115224919_add_round_types.rb

```diff
@@ -0,0 +1,6 @@
+class AddRoundTypes < ActiveRecord::Migration[5.2]
+  def up
+    Roundtype.create(name:'swiss')
+    Roundtype.create(name:'elimination')
```

Comment:
Can you just put these in the create_rounds migration? I'd prefer to not make new migrations unless you're changing one that's already deployed to production?